aws/resource_aws_launch_template.go

```diff
@@ -563,7 +563,7 @@ func resourceAwsLaunchTemplateRead(d *schema.ResourceData, meta interface{}) err
 		return err
 	}
 
-	if strings.HasPrefix(aws.StringValue(ltData.InstanceType), "t2") {
+	if strings.HasPrefix(aws.StringValue(ltData.InstanceType), "t2") || strings.HasPrefix(aws.StringValue(ltData.InstanceType), "t3") {
```

Comment:
if there's more here in the future it could be worth regex-ing this?

Reply:
If/when there is another instance type that requires the credit specification, we may want to consider doing something that is like:

```go
// if it is something like t4.X
if strings.HasPrefix(aws.StringValue(ltData.InstanceType), "t") && !strings.HasPrefix(aws.StringValue(ltData.InstanceType), "t1.")
// if it is something different, something more complex like regex or slice based matching sounds good!
```

The explicit nature of not having `t3` immediately available did surface some potential bugs in `aws_instance` handling during explicit `t3` testing, so it was actually nice to have it "gated". 🤔 I'm more than happy to follow this up if we think we should!